L1Trigger/L1THGCal/python/customTriggerCellSelect.py

```diff
@@ -29,3 +33,28 @@ def custom_triggercellselect_bestchoice(process,
     parameters = best_conc_proc.clone(NData = triggercells)
     process.hgcalConcentratorProducer.ProcessorParameters = parameters
     return process
+
+
+def custom_coarsetc_onebitfraction(process,
+                                             oneBitFractionThreshold = coarsetc_onebitfraction_proc.oneBitFractionThreshold,
+                                             oneBitFractionLowValue = coarsetc_onebitfraction_proc.oneBitFractionLowValue,
+                                             oneBitFractionHighValue = coarsetc_onebitfraction_proc.oneBitFractionHighValue,
+                                             ):
+    parameters = coarsetc_onebitfraction_proc.clone()
+    parameters.oneBitFractionThreshold = oneBitFractionThreshold;
+    parameters.oneBitFractionLowValue = oneBitFractionLowValue;
+    parameters.oneBitFractionHighValue = oneBitFractionHighValue;
+
+    return process
+
+
+
+
+def custom_coarsetc_equalshare(process,
+                                         nTriggerCellsForDivision = 4
```

Comment:
Indentation to be fixed.
Default value should follow the same format as other custom functions, using values from existing `PSet` to avoid having different default values in different files.